features/step_definitions/user_steps.rb

```diff
@@ -256,6 +256,14 @@ def get_series_name(age, classname, name)
   step %{I should not see "#{title}"}
 end
 
+When /^I change my username to "([^\"]*)"/ do |new_name|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.